README.md

```diff
@@ -1 +1,33 @@
-# SeleniumProject
\ No newline at end of file
+# SeleniumProject
+
+## Introduction
+This is my first test automation project based on Selenium-Webdriver with Python. It's still developing package of
+automated tests of skleptest.pl demo website. The collection of tests contains:
+* Register user test
+* Login user tests
+* Adding item do cart
+
+## Project Structure
+Here you can find a short description of main directories, and it's content
+
+[locators](locators) - there are locators of web elements in locators.py grouped in classes\
+[pages](pages) - there are sets of method for each test step\
+[tests](tests) - there are sets of tests for main functionalities of website
+
+## Getting Started
+To enjoy the automated tests, develop the framework or adapt it to your own purposes, just download the project or clone
+repository. You need to install packages using pip according to requirements.txt file. Run the command below in
+terminal:
```

Comment:
Przed tym zaproponował bym zbudowanie wirtualnego środowiska, np. w zależności od pythona jakiego masz:
```
python3.10 -m venv .venv
source .venv/bin/activate
```
Pamiętaj tylko o dodaniu do `.gitignore` tego folderu `.venv`. To co podałem jest dla Linuxa. Dla Windowsa jest może troche innaczej (no i musisz mieć to zainstalowane na kompie https://docs.python.org/3/library/venv.html)